HealthKitReporter/Classes/Model/Payload/Quantity.swift

```diff
@@ -118,6 +123,7 @@ extension Quantity: Payload {
         from dictionary: [String : Any]
     ) throws -> Quantity {
         guard
+            let uuid = UUID(uuidString: dictionary["uuid"] as! String),
```

Comment:
No-no on force unwrapping (this will be not the case as soon the uuid will be just a String)

```suggestion
            let uuid = dictionary["uuid"] as? String,
```